app/javascript/mastodon/features/account_timeline/index.js

```diff
@@ -73,6 +75,12 @@ class AccountTimeline extends ImmutablePureComponent {
     this.props.dispatch(expandAccountTimeline(accountId, { withReplies }));
   }
 
+  componentDidMount () {
+    if (this.props.params.accountId === me) {
+      this.disconnect = this.props.dispatch(connectUserTimelineStream(me));
```

Comment:
I've just realized this should not be necessary because the user stream is always active, it starts when the app is opened.